document-generator-company-report/src/main/java/uk/gov/companieshouse/document/generator/company/report/service/PscsService.java

```diff
@@ -0,0 +1,46 @@
+package uk.gov.companieshouse.document.generator.company.report.service;
+
+import javax.sql.rowset.serial.SerialException;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import org.springframework.web.util.UriTemplate;
+import uk.gov.companieshouse.api.ApiClient;
+import uk.gov.companieshouse.api.error.ApiErrorResponseException;
+import uk.gov.companieshouse.api.handler.exception.URIValidationException;
+import uk.gov.companieshouse.api.handler.psc.request.PscsList;
+import uk.gov.companieshouse.api.model.psc.PscApi;
+import uk.gov.companieshouse.api.model.psc.PscsApi;
+import uk.gov.companieshouse.document.generator.company.report.exception.ServiceException;
+
+@Service
+public class PscsService {
+
+    @Autowired
+    private CompanyReportApiClientService companyReportApiClientService;
+
+    private static final UriTemplate GET_PSCS_URI =
+            new UriTemplate("/company/{companyNumber}/persons-with-significant-control");
+
+    public PscsApi getPscs(String companyNumber) throws ServiceException {
+
+        PscsApi pscsApi;
+
+        ApiClient apiClient = companyReportApiClientService.getApiClient();
+
+        String uri = GET_PSCS_URI.expand(companyNumber).toString();
+
+        try {
+
+            PscsList pscsList = apiClient.pscs().list(uri);
+            pscsList.addQueryParams("items_per_page", "100");
+
+            pscsApi = pscsList.execute().getData();
```

Comment:
could have this as `return pscsList.execute().getData();` then can remove the model declaration at the top and the return below?

I think we also could do this for the other services, like officers, etc.